episodes/infrastructure.md

```diff
@@ -168,6 +168,8 @@ These are:
 We will address all of the files later in the training.
 For now, we will move on to complete the basic setup of the lesson.
 
+
```

Comment:
Add `.Rproj` file to the list of repository files above - GitHub won't let me make suggestions to the above lines so leaving this as a comment.

Reply:
I'm going to leave this out because I do not think that the `Rproj` file really fits the description of "accompanying files primarily intended for the repository itself rather than the lesson website" (nor is it exactly "intended for the lesson website" but instead lies somewhat ambiguously in-between.

Reply:
Yes, I was wondering about that as well as was not sure is it better to be consistent if not 100% correct as we mention this file under "Configuring a Lesson Repository" 🙈 😄 @tobyhodges .